package/Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM registry.suse.com/bci/golang:1.22 AS golang
 
 # Builder and xx only need to support the host architecture.
 FROM --platform=$BUILDPLATFORM rancher/mirrored-tonistiigi-xx:1.3.0 AS xx
-FROM --platform=$BUILDPLATFORM registry.suse.com/bci/golang:1.22 AS builder
+FROM --platform=$BUILDPLATFORM golang AS builder
```

Comment:
Out of curiosity, why do we need to change from BCI?

Reply:
I didn't, this was redundant. It is referring to the line `3` above:
```
FROM registry.suse.com/bci/golang:1.22 AS golang
```

The PR review not showing it makes it confusing.

Reply:
Ohh, I see. Thanks for clarifying.

Reply:
However, it turns out that forces cross emulation instead of cross compilation, so I will remove it.